tests/conftest.py

```diff
@@ -94,6 +94,24 @@ def vault(pm, gov, rewards, guardian, management, token):
     vault.setManagement(management, {"from": gov})
     yield vault
 
+@pytest.fixture(scope ="module")
+def deploy_strategy_proxy(
+    deploy_angle_voter,
+    gov,
+    AngleStrategyProxy
+):
+    strategy_proxy = gov.deploy(AngleStrategyProxy, deploy_angle_voter.address)
+    yield strategy_proxy
+
+
+@pytest.fixture(scope ="module")
+def deploy_angle_voter(
```

Comment:
Recommend changing fixture names for this + above to just 'angle_voter' and 'strategy_proxy'

Reply:
Done